vm/actor/src/builtin/multisig/types.rs

```diff
@@ -22,6 +22,12 @@ impl TxnID {
     }
 }
 
+/// Proposal Hash
+// Blake2b 32 hash 
+#[derive(Clone, Copy, Default, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct ProposalHash(pub [u8; 32]);
```

Comment:
This will serialize incorrectly (as an array of integers not as a slice of bytes) should tag with serde_bytes

Reply:
I had something similar already in my PR. I dont see the need to create a separate type though

Reply:
@nannick Can I use your branch then ? I just needed this change so it is compatible with lotus.

Reply:
we delete branches and they get squashed, so the commit hash will be invalid once merged, though